docker/prod/uwsgi/uwsgi.service

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I think it would be good to also setup the beta env here. I know it is extra work, but I really wish I would've done that in the first place with LB. This way we can initially deploy to hetzner with beta env and then later promote it to prod -- during this process we shake out the issues of running two setups on the same servers.